README.md

```diff
@@ -43,8 +43,19 @@ from float8_experimental.fsdp_utils import precompute_float8_dynamic_scale_for_f
 # create model
 m = Model(...)
 
+# optional: filter layers from being eligible for float8 conversion
+def layer_filter_fn(fqn: str, mod: torch.nn.Module):
```

Comment:
nit: how do you feel about using "module" instead of "layer", which may be more canonical in PyTorch?

Reply:
sgtm